src/test/js/multibranch/multibranchOpening.js

```diff
@@ -0,0 +1,140 @@
+const jobName = 'withGitFlow';
+const path = require("path");
+const pathToRepo = path.resolve('./target/test2-project-folder');
+const soureRep = './src/test/resources/multibranch_2';
+const git = require("../../../main/js/api/git");
+
+
+module.exports = {
+
+
+    // ** creating a git repo */
+    before: function (browser, done) {
+          // we creating a git repo in target based on the src repo (see above)
+          git.createRepo(soureRep, pathToRepo)
+              .then(function () {
+                  git.createBranch('feature/1', pathToRepo)
+                      .then(done);
+              });
+    },
+    
+    /**
+     * Make sure we can open the master branch results screen from activity
+     */
+    'open master branch from activity': function (browser) {
+
+         var jobName = "masterActivityMB";      
+         var multibranchCreate = browser.page.multibranchCreate().navigate();      
+         multibranchCreate.createBranch(jobName, pathToRepo);
+
+         var blueActivityPage = browser.page.bluePipelineActivity().forJob(jobName, 'jenkins');
+
+         blueActivityPage.waitForElementVisible('tr[id^="master"]');
+         blueActivityPage.click('tr[id^="master"]');
+         
+         //check results look kosher:
```

Comment:
The following block should be moved to the `blueActivityPage` to a encapsulated function e.g. `isMultiBranchKosher` and the we can do `blueActivityPage.isMultiBranchKosher()` and not repeat this code.